polysh/buffered_dispatcher.py

```diff
@@ -68,10 +75,11 @@ def readable(self):
 
     def writable(self):
         """Do we have something to write?"""
-        return self.write_buffer != ''
+        return self.write_buffer != b''
 
     def dispatch_write(self, buf):
         """Augment the buffer with stuff to write when possible"""
+        assert type(buf) == bytes
```

Comment:
`isinstance()` would be better.  We may want to subclass `bytes` without touching all occurrences.

Reply:
Done